nixos/modules/system/boot/binfmt.nix

```
@@ -30,6 +30,7 @@ let
 
   getEmulator = system: (lib.systems.elaborate { inherit system; }).emulator pkgs;
```

What @Ericson2314 probably means is to simply optionally do this:

```suggestion
  getEmulator = system: (lib.systems.elaborate { inherit system; }).emulator pkgsStatic;
```